modules/universal-robots-client-library/1.5.0/source.json

```diff
@@ -0,0 +1,10 @@
+{
+    "url": "https://github.com/UniversalRobots/Universal_Robots_Client_Library/archive/refs/tags/1.5.0.tar.gz",
+    "integrity": "sha256-sXk5vqpmqubhCGWJuybgoUuZc6txo822noYcU9gQ0mA=",
+    "strip_prefix": "Universal_Robots_Client_Library-1.5.0",
+    "patch_strip": 1,
+    "patches": {
+        "add_build_file.patch": "sha256-fCil/zaqY/C8+NIER4Ch8Igk+puyZN/Z6R+6I9d8DRU=",
```

Comment:
Did this patch file change? If not, could you replace it with a symlink to the old one so that the diff is smaller?

Reply:
done